release-notes/opensearch-release-notes-2.12.0.md

```diff
@@ -0,0 +1,1306 @@
+<h1>OpenSearch and OpenSearch Dashboards 2.12.0 Release Notes</h1>
+<h2>RELEASE HIGHLIGHTS</h2>
+<ul>
+OpenSearch 2.12.0 includes a number of features designed to increase performance for search and analytics applications as well as user experience enhancements and several new and enhanced machine learning tools. Experimental functionality includes the OpenSearch Assistant Toolkit for building interactive user experiences and new cross-cluster monitors. OpenSearch upgrades to Apache Lucene 9.9.2 with this release. 
+</ul>
+
+<h2>NEW FEATURES</h2>
+<ul>
+<li>Integration with Apache Spark lets you analyze all of your operational data in a single place using OpenSearch in combination with Apache Spark. </li>
+<li>Conversational search is generally available, providing comprehensive functionality to build conversational experiences using OpenSearch’s lexical, vector, and hybrid search features.</li>
+<li>A new workflow engine lets you automate configurations for ML Commons resources, allowing you to set up machine learning resources to support AI use cases without the need to manually create resources or write custom scripts.</li>
```

Comment:
This item (new workflow engine) should be moved to the Experimental section below. Per docs, now seeing that this feature is experimental. https://opensearch.org/docs/latest/automating-configurations/index/